data/editor.config

```diff
@@ -1212,16 +1221,33 @@
         <include field="operator" />
         <include field="height" />
       </type>
+      <type id="man_made-flare">
+        <include field="name" />
+        <include field="operator" />
+        <include field="height" />
+      </type>
+      <type id="man_made-petroleum_well">
+        <include field="name" />
+        <include field="operator" />
+      </type>
       <type id="man_made-water_well">
         <include field="name" />
         <include field="operator" />
       </type>
-      <!-- Uncomment after a map style is added
+      <type id="man_made-tower">
+        <include field="name" />
+        <include field="operator" />
+        <include field="height" />
+      </type>
       <type id="man_made-water_tower">
         <include field="name" />
         <include field="operator" />
         <include field="height" />
-      </type>-->
+      </type>
+      <type id="man_made-communications_tower">
+        <include group="poi_internet" />
```

Comment:
Treat comm tower like POI?

Reply:
These towers are huge and are POIs definitely https://wiki.openstreetmap.org/wiki/Tag:man_made%3Dcommunications_tower

Reply:
Well, POI is about name, address, opening hours, contact*, phone, ...

Reply:
You're right! I'll change it